examples/mqtt-service-subscribe-message/mqtt_service_subscribe_message.md

```diff
@@ -0,0 +1,18 @@
+# MQTT service - Subscribe to messages
+
+The `mqtt:Service` connects to a given MQTT server via the `mqtt:Listener`, and allows to receive messages from different topics. Once subscribed to a topic, when new messages are received, the `onMessage` method gets invoked. If there are errors when invoking the method, the `onError` remote method will be invoked with the relevant `error`. If the method is not implemented, the error will be logged to the console. Use this to receive messages from a set of topics in an MQTT server.
```

Comment:
```suggestion
The `mqtt:Service` connects to a given MQTT server via the `mqtt:Listener`, and allows to receive messages from different topics. Once new messages are received from a subscribed topic, the `onMessage` method gets invoked. If there are errors when invoking the method, the `onError` remote method will be invoked with the relevant `error`. If the method is not implemented, the error will be logged to the console. This can be used to receive messages from a set of topics in an MQTT server.
```